substrate/frame/src/lib.rs

```diff
@@ -329,6 +330,8 @@ pub mod testing_prelude {
 
 	pub use sp_io::TestExternalities as TestState;
 
+	pub use sp_io::storage::*;
```

Comment:
I don't really like this reexport. I think `storage::clear` is way more meaningful than just `clear`, and same for `set` and `append`.

Reply:
roger, i just replaced it with `pub use sp_io::storage;`